src/metainfo/yast2-tune.metainfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Dominique Leuenberger <dimstar@opensuse.or> -->
```

Comment:
Glad that you know my email address :) Will fix this of course (and finally buy a new keyboard.. my 'g' keeps on being missed